api/routes/subcriterias/get_sub_criteria.py

```diff
@@ -254,15 +269,25 @@ def map_single_field_answer(theme: list, questions: dict) -> str:
 
 def map_application_with_sub_criteria_themes(
     application_id: str, theme_id: str, fund_id: str, round_id: str
+):
+    themes_fields = get_themes_fields(theme_id, fund_id, round_id)
+    current_app.logger.info("mapping subcriteria contents")
+    return map_application_with_sub_criteria_themes_fields(
+        application_id,
+        theme_id,
+        themes_fields,
+    )
+
+
+def map_application_with_sub_criteria_themes_fields(
+    application_id: str, theme_id: str, themes_fields: list[dict]
 ):
     """function maps answers from application with assessor task list
     themes through field ids.
     Args: application_id, theme_id, fund_id, round_id
     Exceptions: returning custom exception along with openapi
     validation detail.
     """
-
-    themes_fields = get_themes_fields(theme_id, fund_id, round_id)
```

Comment:
how comes we're removing themes_fields from this func?

Reply:
Hey!  The signature of the method has changed, so we pass `themes_fields` in, was split into another function so we could re-use this functionality with the `all_uploaded_documents` functionality!